extensions/wikia/ImageReviewMercury/ImageReviewMercuryController.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class ImageReviewMercuryController extends WikiaSpecialPageController {
+
+	private $imageReview;
+	public function __construct() {
+
+		$this->imageReview = (new ImageReviewMercury);
+		parent::__construct( 'ImageReviewMercury' );
+	}
+
+	public function index() {
+		$title = Title::newFromText( 'ImageReviewMercury', NS_SPECIAL );
+
+		$this->wg->Title = $title;
```

Comment:
Perhaps it's worth to simply use `$this->wg->Title = Title::newFromText( 'ImageReviewMercury', NS_SPECIAL );` as there is nothing done with the variable apart from yet another assignment?